server/routes/visitJourney/checkYourBooking.ts

```diff
@@ -74,27 +73,6 @@ export default class CheckYourBooking {
         operationId: res.locals.appInsightsOperationId,
       })
 
-      if (isUpdate) {
```

Comment:
The actual 'booking' stage at the end is now the same for first-time booking or updating existing booking.